pf4j/src/main/java/ro/fortsoft/pf4j/util/FileUtils.java

```diff
@@ -96,27 +92,27 @@ public static boolean delete(File fileOrFolder) {
 		return success;
 	}
 
-	public static List<File> getJars(File folder) {
+	public static List<File> getJars(Path folder) {
```

Comment:
I am OK with the modifications from this class. My intention is to move this code to Java 7 NIO API, and to use `Files.newDirectoryStream` (less code lines and probably faster).